Description: Always specify rpath for private libraries
Author: Jelmer Vernooij <jelmer@samba.org>
Last-Update: 2012-02-24
Applied-Upstream: no

=== modified file 'buildtools/wafsamba/samba_utils.py'
--- old/buildtools/wafsamba/samba_utils.py	2012-02-12 17:44:09 +0000
+++ new/buildtools/wafsamba/samba_utils.py	2012-02-24 14:11:05 +0000
@@ -79,7 +79,7 @@
     ret = set()
     if bld.env.RPATH_ON_INSTALL:
         ret.add(bld.EXPAND_VARIABLES(bld.env.LIBDIR))
-    if bld.env.RPATH_ON_INSTALL_PRIVATE and needs_private_lib(bld, target):
+    if bld.env.RPATH_ON_INSTALL_PRIVATE:
         ret.add(bld.EXPAND_VARIABLES(bld.env.PRIVATELIBDIR))
     return list(ret)
 

